controllers/component_build_controller_pac.go

```diff
@@ -871,6 +871,11 @@ func (r *ComponentBuildReconciler) ConfigureRepositoryForPaC(ctx context.Context
 		IsAppInstallationExpected: true,
 	})
 	if err != nil {
+		if strings.Contains(err.Error(), "installation has been suspended") {
```

Comment:
Does it make sense to check the condition inside the git package like for other errors, e.g. [this](https://github.com/redhat-appstudio/build-service/blob/main/pkg/git/github/github_app.go#L61)?

Reply:
403 is general forbidden, so I'd rather check error message

Reply:
Yes, we definitely need to check error message, my point was in the place of the check.

Reply:
ok I'll check in github package